var/spack/repos/builtin/packages/nemsio/package.py

```diff
@@ -18,11 +18,16 @@ class Nemsio(CMakePackage):
 
     maintainers = ['t-brown', 'edwardhartnett', 'kgerheiser', 'Hang-Lei-NOAA']
 
+    variant('mpi', default=True, description='Build Nemsio with MPI')
+    # Nemsio 2.5.3 and below require MPI
+    conflicts('-mpi', when='@:2.5.3')
```

Comment:
Can you switch `-mpi` with `~mpi`? `-` can be ambiguous (in this case I don't think that's an actual problem, but I think it keeps things clear to use `~`.

Reply:
Fixed. Curious where the ambiguity comes from. I saw in the documentation that `~` can be ambiguous when use on the command line because it also means home directory.

Reply:
The ambiguity for `-` in spack specs can be an issue because it can appear as part of a spec name: so if you have a package called `FooBar` (as defined in the package.py`) you could do `spack install foo-bar`, and if that package has a boolean variant `x` then you can do `spack install foo-bar~x` .